server/models/postgis/statuses.py

```diff
@@ -57,6 +57,7 @@ class ValidatingNotAllowed(Enum):
     USER_NOT_VALIDATOR = 100
     USER_NOT_ACCEPTED_LICENSE = 101
     USER_NOT_ON_ALLOWED_LIST = 102
+    PROJECT_NOT_PUBLISHED = 103
```

Comment:
These are defined in the postgis directory...does that mean we will need to add a database upgrade?

Reply:
I'd be interested to know as well. I thought it was an ORM only thing but it looks like postgreSQL uses theses value internally. Most probably to add a constraint on the possible values.
What I don't get though is that we never use this class for any table in the model. Looks like there's no need for an Enum here.
Oh wait! This is not an `SQLAlchemy Enum`. It's from the python core library.
No database migration required, IMHO.